### PR TITLE
016/master

### PR DESCRIPTION
**Pull request by BenGitsCode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,13 +249,13 @@ Consider the following Ember Class:
...
+let bob = Person.create({
+  givenName: 'Bob',
   surname: 'Belcher'
 })
 ```
 
-Suppose we wanted `tina` to have a property called `fullName` which was equal
+Suppose we wanted `bob` to have a property called `fullName` which was equal
  to his given name plus his surname.
 We could obviously define a function in the Class definition to return that
```
Comment (bernardlee) [inline README.md]: "obviously" seems like a word that should be avoided.

Comment (BenGitsCode) [reply README.md]: I agree with that. And obviously `tina` should be `bob`, obviously.

Comment (BenGitsCode) [reply README.md]: This shows as bob on line 258, weird github side effect?

Comment (bernardlee) [reply README.md]: > Suppose we wanted `bob` ...

Obviously.  It shows up on line 258 when I open the file in Atom.

Comment (BenGitsCode) [reply README.md]: a cat of what I just pushed: 

```md
Suppose we wanted `bob` to have a property called `fullName` which was equal
 to his given name plus his surname.
We could now define a function in the Class definition to return that
 value.

```

**Pull request merged**